proto/osmosis/lockup/tx.proto

```diff
@@ -43,7 +43,9 @@ message MsgUnlockTokensResponse {
 }
 
 message MsgUnlockPeriodLock {
-	uint64 ID = 1;
+	bytes owner = 1 
```

Comment:
This is for the signature check?  Should we allow anyone to unlock someone else's tokens?

Reply:
yes, it's for signature check. unlocking other's token is disabled now.